ui/js/directives/treeherder/bottom_nav_panel.js

```diff
@@ -34,67 +34,3 @@ treeherder.directive('thRelatedBugQueued', function () {
         templateUrl: 'partials/main/thRelatedBugQueued.html'
     };
 });
-
-treeherder.directive('thFailureClassification', [
```

Comment:
Unused as of https://github.com/mozilla/treeherder/commit/4b9cb14a#diff-a1e0123d56c83d6825063acef3105ffcL16